_posts/2019-05-29-cockpit-195.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Cockpit 195
+author: lis
+date: 2019-05-29
+tags: cockpit linux
+slug: cockpit-195
+category: release
+summary: Firewall, storage and machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 195.
+
+### Simplified support for external TLS termination
+
+It's now easier to configure Cockpit's web server `cockpit-ws` to run
+behind a
+[TLS termination proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy).
+If the proxy runs on the same machine, then `cockpit-ws` can be run with
+the new `--for-tls-proxy` option, which will adjust the allowed
+[Origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin)
+and
+[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+to http:// URLs. With this option, it's no longer necessary to
+explicitly configure
+[cockpit.conf](https://cockpit-project.org/guide/latest/cockpit.conf.5.html).
+
+### Firewall: More support for zones
+
+In addition to the support for adding services to specific zones in the
+last release, the firewall page now allows the adding and removing of
+the zones themselves.
+
+![Adding firewall zones](/images/firewall-add-zone.png)
+
+### Machines: Additional storage options
+
+During virtual machine creation it is now possible to select the
+storage pool to use for the machine.
+
+![Select storage pool during machine creation](/images/machines-select-storage-pool.png)
+
+It is now also possible to create a storage pool using a physical disk
+device as the backing store.
+
+![Create storage pool on physical disk device](/images/machines-new-pool-physical-disk-device.png)
+
+Finally, it is now possible to configure the disk cache policy when
+adding storage devices to virtual machines.
+
+![Adjust virtual machine disk cache policy](/images/machines-disk-cache-policy.png)
+
+### Machines: Support for changing memory allocation
+
+The machines page also gained support for changing the current and
+maximum memory allocations of a virtual machine.
+
+![Adjust virtual machine memory allocation](/images/machines-memory-allocation.png)
+
+### Storage: Select encryption type separately from filesystem
+
+The selection of the filesystem type and filesystem encryption options
+have been uncoupled on the storage page, allowing for the use of LUKS
+with filesytems other than ext4 and xfs.
+
+![New encryption option](/images/storage-separate-encryption-option.png)
+
+### Try it out
+
+Cockpit 195 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/195)
+ * [Fedora 29](https://bodhi.fedoraproject.org/updates/cockpit-195-1.fc29)
```

Comment:
We just [stopped doing that](https://github.com/cockpit-project/cockpit/pull/11951), so please drop that line.